src/node/node_state.h

```diff
@@ -478,6 +478,11 @@ namespace ccf
 
       pal::PlatformAttestationReportData report_data =
         crypto::Sha256Hash((node_sign_kp->public_key_der()));
+      LOG_INFO_FMT("public_key={}", ds::to_hex(node_sign_kp->public_key_der()));
+      LOG_INFO_FMT(
+        "public_key_hash={}",
+        crypto::Sha256Hash((node_sign_kp->public_key_der())).hex_str());
```

Comment:
The public key can be extracted from the certificate written by the node to disk (typically under workspace/$testname_nodeid/nodeid.pem), or obtainable via tls:

node_pubk_hash=$(echo | openssl s_client -showcerts -connect "${stripped_node_address}" 2>/dev/null | openssl x509 -pubkey -noout | openssl ec -pubin -outform der 2>/dev/null | sha256sum | awk '{ print $1 }')